os/tools/memstats_gnueabil.py

```diff
@@ -150,6 +150,63 @@ def printLibrarySizes():
 			print "\t"+str(sizearr[Required.keys().index(r)]),
 		print "\t" + str(librarySize[l]) + "\t" + l
 
+
+def sortByTotal(n):
+        return int(n[len(n)-2])
+
+def sortPrintLibrarySizes():
+        results = []
```

Comment:
The indentations of lines is done with spaces and tabs. Some are indented with spaces and some with tabs. Please follow a single strategy.